lua/neogit/lib/buffer.lua

```diff
@@ -469,7 +470,9 @@ function Buffer.create(config)
   end
 
   if config.after then
-    buffer:call(config.after)
+    buffer:call(function()
+      config.after(buffer, win)
```

Comment:
Just curious, whats the use case for passing the window into this?

Reply:
The buffer may not always be shown, depends on the value of `open` which is sometimes user configurable, and other autocommands. Additionally, autocommands may open other windows

This just makes sure we only operate on the window we intent, and not scroll e.g a `notify-nvim` popup that happened to appear

Reply:
Ahhh, ok, thats a good call. I've had that issue with the notify popups. Never thought too hard about it, but this is definitely better.